modules/caddyhttp/reverseproxy/upstreams.go

```diff
@@ -397,6 +386,19 @@ func (au AUpstreams) GetUpstreams(r *http.Request) ([]*Upstream, error) {
 
 func (au AUpstreams) String() string { return net.JoinHostPort(au.Name, au.Port) }
 
+func resolveIpVersion(versions *IPVersions) string {
```

Comment:
I think we should move this function to just below the IPVersions type (higher in this file) since it's tightly related.

Reply:
Done.